jekyll/_sass/_layout.scss

```diff
@@ -0,0 +1,379 @@
+        }
+
+        .page-link {
+            display: block;
+            padding: 5px 10px;
+
+            &:not(:last-child) {
+                margin-right: 0;
+            }
+            margin-left: 20px;
+        }
+    }
+}
+
+.draft {
+    background-color: $draft-background;
+    padding: 4px;
+    font-weight: bold;
+    font-size: 0.8em;
+    border-radius: 2px;
+}
+
+
+/**
+ * Site footer
+ */
+.site-footer {
+    border-top: 1px solid $grey-color-light;
+    padding: $spacing-unit 0;
+}
+
+.footer-heading {
+    font-size: 18px;
+    margin-bottom: $spacing-unit / 2;
+}
+
+.contact-list,
+.social-media-list {
+    list-style: none;
+    margin-left: 0;
+}
+
+.footer-col-wrapper {
+    font-size: 15px;
+    color: $grey-color;
+    margin-left: -$spacing-unit / 2;
+    @extend %clearfix;
+}
+
+.footer-col {
+    float: left;
+    margin-bottom: $spacing-unit / 2;
+    padding-left: $spacing-unit / 2;
+}
+
+.footer-col-1 {
+    width: -webkit-calc(35% - (#{$spacing-unit} / 2));
+    width:         calc(35% - (#{$spacing-unit} / 2));
+}
+
+.footer-col-2 {
+    width: -webkit-calc(20% - (#{$spacing-unit} / 2));
+    width:         calc(20% - (#{$spacing-unit} / 2));
+}
+
+.footer-col-3 {
+    width: -webkit-calc(45% - (#{$spacing-unit} / 2));
+    width:         calc(45% - (#{$spacing-unit} / 2));
+}
+
+@include media-query($on-laptop) {
+    .footer-col-1,
+    .footer-col-2 {
+        width: -webkit-calc(50% - (#{$spacing-unit} / 2));
+        width:         calc(50% - (#{$spacing-unit} / 2));
+    }
+
+    .footer-col-3 {
+        width: -webkit-calc(100% - (#{$spacing-unit} / 2));
+        width:         calc(100% - (#{$spacing-unit} / 2));
+    }
+}
+
+@include media-query($on-palm) {
+    .footer-col {
+        float: none;
+        width: -webkit-calc(100% - (#{$spacing-unit} / 2));
+        width:         calc(100% - (#{$spacing-unit} / 2));
+    }
+}
+
+
+
+/**
+ * Page content
+ */
+.page-content {
+    padding: $spacing-unit 0;
+}
+
+.page-heading {
+    font-size: 20px;
+}
+
+.post-list {
+    margin-left: 0;
+    list-style: none;
+
+    > li {
+        margin-bottom: $spacing-unit;
+    }
+}
+
+.post-meta {
+    font-size: $small-font-size;
+    color: $grey-color;
+}
+
+.post-link {
+    display: block;
+    font-size: 24px;
+}
+
+.page-content {
+    .grid-1 {
+        display: grid;
+        grid: auto-flow / 1fr;
+        column-gap: 12px;
+    }
+
+    .grid-2 {
+        display: grid;
+        grid: auto-flow / repeat(2, 1fr);
+        column-gap: 12px;
+
+        p > img {
+            width: 100%;
+        }
+    }
+
+    .grid-3 {
+        display: grid;
+        grid: auto-flow / repeat(3, 1fr);
+        column-gap: 12px;
+    }
+
+    .grid-4 {
+        display: grid;
+        grid: auto-flow / repeat(4, 1fr);
+        column-gap: 12px;
+    }
+
+    .grid-5 {
+        display: grid;
+        grid: auto-flow / repeat(5, 1fr);
+        column-gap: 12px;
+    }
+
+    img {
+        background-color: $grey-20;
```

Comment:
We could user `$grey-10` instead of `$grey-20` otherwise components that have a grey 20 background like the message bars are invisible in the assets.

Reply:
Fixed in the next branch.